src/repr/src/row.rs

```diff
@@ -166,6 +166,70 @@ impl Ord for Row {
     }
 }
 
+#[allow(missing_debug_implementations)]
```

Comment:
We should implement `Debug` for `StableRegion`. See https://github.com/frankmcsherry/columnation/issues/13 for details.